src/api-reference/user/index.markdown

```diff
@@ -27,7 +35,7 @@ Name|Type|Format|Description
 -----|------|------|--------------
 `loginID`|`string`|-|The URL-encoded SAP Concur login of the user. Optional.
 
-### Response
+### <a name="getUserSchema"></a>Get User Response Schema
 
```

Comment:
I know you didn't make any changes to this but there is some stuff that is just straight up inaccurate here. Could you please remove the Format from the end of the description into the appropriate Format column? Also, if the format is a Y/N, there's no way the type is a string, please change those to boolean.

Reply:
Fixed